test/tools/libtesteth/ImportTest.cpp

```diff
@@ -80,7 +80,7 @@ bytes ImportTest::executeTest()
 				if(opt.trValueIndex != -1 && opt.trValueIndex != tr.valInd)
 					continue;
 
-				std::tie(tr.postState, std::ignore, tr.changeLog) =
+				std::tie(tr.postState, tr.logs, tr.changeLog) =
```

Comment:
I think it's nicer to receive `ImportTest::ExecOutput` and use the logs in it.  In a separate PRs we can read more from the `ExecOutput`.